package/gluon-web-admin/luasrc/lib/gluon/config-mode/controller/admin/upgrade.lua

```diff
@@ -18,7 +18,7 @@ local unistd = require 'posix.unistd'
 local tmpfile = "/tmp/firmware.img"
 
 
-local function filehandler(meta, chunk, eof)
+local function filehandler(_, chunk, eof)
```

Comment:
While the variable *file* is used across multiple calls of *filehandler*, it can at least be declared local to this file.

Reply:
Add `local file` at the beginning